.jenkins/pytorch/build.sh

```diff
@@ -322,3 +322,18 @@ if [[ "$BUILD_ENVIRONMENT" != *libtorch* && "$BUILD_ENVIRONMENT" != *bazel* ]];
   # don't do this for libtorch as libtorch is C++ only and thus won't have python tests run on its build
   python test/run_test.py --export-past-test-times
 fi
+
+# Test Lazy Tensor Core
+# No particular reasons why this environment, just restrict lazy tensor to one to limit potential breakages.
+if [[ "$BUILD_ENVIRONMENT" == *linux-xenial-cuda11.1* ]]; then
+  pushd lazy_tensor_core/
+  ./scripts/apply_patches.sh
```

Comment:
Having code in the same repo exist as set of patches feels a bit weird
Plus, it puts unreasonable burden on all developers to update those patches when they modify the code
Perhaps `#ifdefs` in the code would be better

Reply:
Those are all good reasons. However, Will has filled up some contexts here. Wondering if you have any suggestions how to make our use case viable.

Reply:
This is OK for now. we can do it better once we refactor the lazy tensor build system for merging lazy tensor to master.